examples/fix_invalid_speakers.py

```diff
@@ -0,0 +1,386 @@
+    for oid, current_speakers in speakers_by_oid.items():
+        corrected_speakers = []
+        for speaker in current_speakers:
+            if speaker.email not in corrections:
+                logger.debug(f'"{speaker.email}" not in CSV. Skipping...')
+                corrected_speakers.append(speaker)
+                continue
+            correction = corrections[speaker.email]
+            if (
+                not correction.corrected_email
+                and not correction.corrected_id
+                and not correction.corrected_name
+            ):
+                logger.debug(f'No corrections for "{speaker.email}". Skipping...')
+                corrected_speakers.append(speaker)
+                continue
+
+            corrected_email = speaker.email
+            if correction.corrected_email == 'DELETE':
+                corrected_email = ''
+            elif correction.corrected_email:
+                corrected_email = correction.corrected_email
+            corrected_id = speaker.id
+            if correction.corrected_id == 'DELETE':
+                corrected_id = ''
+            elif correction.corrected_id:
+                corrected_id = correction.corrected_id
+            corrected_name = speaker.name
+            if correction.corrected_name == 'DELETE':
+                corrected_name = ''
+            elif correction.corrected_name:
+                corrected_name = correction.corrected_name
+            if any((corrected_email, corrected_id, corrected_name)):
+                corrected_speakers.append(
+                    Speaker(corrected_email, corrected_id, corrected_name)
+                )
+        if current_speakers != corrected_speakers:
+            corrected_speakers_by_oid[oid] = corrected_speakers
+
+    for oid, corrected_speakers in corrected_speakers_by_oid.items():
+        payload = {
+            'speaker_email': '|'.join(spk.email for spk in corrected_speakers),
+            'speaker_id': '|'.join(spk.id for spk in corrected_speakers),
+            'speaker': '|'.join(spk.name for spk in corrected_speakers),
+        }
+        if apply:
+            if oid[0] == 'c':
+                payload['channel_oid'] = oid
+                url = 'settings/defaults/metadata/edit/'
+            else:
+                payload['oid'] = oid
+                url = 'medias/edit/'
+            msc.api(url, method='post', data=payload)
+            logger.info(
+                f'Media "{oid}" has been updated: '
+                f'old_speakers={speakers_by_oid[oid]}, new_speakers={corrected_speakers}'
+            )
+        else:
+            logger.info(
+                f'[Dry-run] Media "{oid}" would have been updated: '
+                f'old_speakers={speakers_by_oid[oid]}, new_speakers={corrected_speakers}'
+            )
+
+
+class ArgumentParser(argparse.ArgumentParser):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.add_argument(
```

Comment:
I think the arguments list should be defined after the instantiation to make clear that this class override is only for usage printing.